service/src/main/java/uk/gov/hmcts/reform/fpl/service/translations/provider/TranslatableHearingFurtherEvidenceProvider.java

```diff
@@ -0,0 +1,78 @@
+package uk.gov.hmcts.reform.fpl.service.translations.provider;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.HearingFurtherEvidenceBundle;
+import uk.gov.hmcts.reform.fpl.model.SupportingEvidenceBundle;
+import uk.gov.hmcts.reform.fpl.model.common.DocumentReference;
+import uk.gov.hmcts.reform.fpl.model.common.Element;
+import uk.gov.hmcts.reform.fpl.model.interfaces.TranslatableItem;
+import uk.gov.hmcts.reform.fpl.service.translations.provider.decorator.SupportingEvidenceBundleTranslatorDecorator;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+import static java.util.Collections.unmodifiableList;
+import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.element;
+
+@Component
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class TranslatableHearingFurtherEvidenceProvider implements TranslatableListItemProvider {
+
+    private static final String HEARING_FURTHER_EVIDENCE_DOCUMENTS_KEY = "hearingFurtherEvidenceDocuments";
+
+    private final SupportingEvidenceBundleTranslatorDecorator decorator;
+
+    @Override
+    public List<Element<? extends TranslatableItem>> provideListItems(CaseData caseData) {
+        return unmodifiableList(getElements(caseData));
+    }
+
+    private List<Element<SupportingEvidenceBundle>> getElements(CaseData caseData) {
+        return caseData.getHearingFurtherEvidenceDocuments()
+            .stream()
+            .flatMap(x -> x.getValue().getSupportingEvidenceBundle().stream())
+            .collect(Collectors.toList());
+    }
+
+    @Override
+    public DocumentReference provideSelectedItemDocument(CaseData caseData, UUID selectedOrderId) {
+        return getElements(caseData)
```

Comment:
there is a method in ElementUtils called `findElement` that may be useful for these methods